SoftLayer/managers/block.py

```diff
@@ -510,16 +510,20 @@ def cancel_block_volume(self, volume_id,
         block_volume = self.get_block_volume_details(
             volume_id,
             mask='mask[id,billingItem[id,hourlyFlag]]')
-        billing_item_id = block_volume['billingItem']['id']
+        billingItem = 'billingItem' in block_volume
+        if billingItem is False:
+            print('The block storage was cancelled')
```

Comment:
The managers should not print out anything. I'd rather we raise an exception here.
https://github.com/softlayer/softlayer-python/blob/master/SoftLayer/managers/hardware.py#L87 has a good example of how to handle this sort of logic.